LL/eoganachta_casil.ttl

```diff
@@ -1821,6 +1822,7 @@
     owl:sameAs <#Sechnasaig>.
 
 <#Rectabrat-22262fbc>
+    a irishTitles:Rí ;
     owl:sameAs <#Rechtabrat>;
     foaf:title "rig"@sga, "king"@en;
     foaf:nick "ind rige"@sga, "the king"@en;
```

Comment:
You just stated that you were happy with "ind rige" to mean that he was king so I am rather confused.  I will update Fiachra.